tests/test_vcf_examples.py

```diff
@@ -448,9 +449,48 @@ def test_call_LAA(self, ds):
             [[1, 2, 3], [2, 3, -2]],
             [[1, -2, -2], [1, -2, -2]],
             [[2, -2, -2], [1, -2, -2]],
+            [[1, -2, -2], [-1, -2, -2]],
+            [[2, -2, -2], [2, -2, -2]],
         ]
         nt.assert_array_equal(ds.call_LAA.values, call_LAA)
 
+    def test_call_LPL(self, ds):
+        call_LPL = [
+            [
+                [100, 0, 105, -2, -2, -2, -2, -2, -2, -2],
+                [0, 100, 200, -2, -2, -2, -2, -2, -2, -2],
+            ],
+            [
+                [0, 100, 200, -2, -2, -2, -2, -2, -2, -2],
+                [154, 22, 0, -2, -2, -2, -2, -2, -2, -2],
+            ],
+            [
+                [1002, 55, 1002, 0, 55, 1002, -2, -2, -2, -2],
```

Comment:
The inner dimension is 10 because at POS 3351, the first sample has a diploid genotype and three alternate allele indices. (I have pasted the test data in this pull request's description for easy reference.) Therefore, the LPL field should have ten entries, one for each possible genotype from the alleles, including the reference allele. In this case, all of the values are the missing value because the entry does not have a PL field.

I think that if we reduced the size by truncating the data, it would be more complicated for data consumers. We also need to support the case where only some, but not all of the PL values are missing, right? If so, I can add a test case for this.